pom.xml

```
@@ -24,6 +24,7 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
     <!-- dependencies -->
+    <dgclib.version>0.3.0-4fe70cd</dgclib.version>
```

```suggestion
    <dgclib.version>0.3.1</dgclib.version>
```
dgc-lib now has a final release with connector